spec/rubocop/cop/rails/avoid_duplicate_ignored_columns_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::AvoidDuplicateIgnoredColumns, :config do
+  context 'with no duplicate self.ignored_columns= call' do
```

Comment:
That's just my two cents:

```suggestion
  context 'with no duplicate `self.ignored_columns= call`' do
```